server_installation/topics/operator/keycloak-cr.adoc

```diff
@@ -29,7 +29,7 @@ ifeval::[{project_community}==true]
  or an Ingress for Kubernetes
 endif::[]
  for the {project_name} cluster.
-* `externalDatabase` - applies only if you want to connect an externally hosted database. That topic is covered in the xref:_external_database[external database] section of this guide.
+* `externalDatabase` - in order to connect to an externally hosted database. That topic is covered in the xref:_external_database[external database] section of this guide. Setting it to false should be used only for testing purposes and will install an embedded PostgreSQL database. Be aware that externalDatabase:false is *NOT* supported in prod environments.
```

Comment:
change "prod" to "production"
Not clear what action is what "will install an embedded PostgreSQL database"  - setting it to False?